guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+
+#### Edit Mint URL & Refresh Mint Settings
+
+{% include image-gallery.html pages = page.images_mint-url-edit %}
+
+It's useful to provide users with the ability to manually update or refresh mint settings in their wallets. While best practices suggest that wallets should auto-refresh and update mint settings periodically, there may be cases where this isn't supported, or a mint undergoes significant updates, such as a URL change. By allowing users to edit the mint URL or refresh settings manually, you ensure that users always have the latest mint configurations.
+
+### Security and Privacy
+
+#### Backup and Restore for Cashu
+The Cashu backup and restoration process is designed to ensure users can securely recover their wallets and maintain access to their bitcoin-backed ecash tokens in the event they switch devices or experience data loss. **Users should only use their recovery seed phrase once.** Repeated use of the seed phrase for restoration can lead to synchronization issues and potential errors. This is due to the fact that each time you restore, you might be dealing with an outdated state of the wallet, which can cause discrepancies in token balances and transactions.
+
+{% include tip/open.html color="blue" icon="info" label="Single Use Recovery" %}
+
+After successfully restoring your wallet using a recovery seed phrase, it is highly recommended that you generate a new recovery seed phrase immediately. This step is crucial because the original recovery phrase has now been used and could be more susceptible to synchronization risks.
+
+{% include tip/close.html %}
+
+#### Backup Process
+{% include image-gallery.html pages = page.images_backup %}
+
+1. **Deterministic Wallet with Seed Phrase**: Cashu uses a deterministic wallet model, where all cryptographic keys and bitcoin-backed ecash tokens can be derived from a single seed phrase. This seed phrase can be generated when the wallet is first created, or a at any point in time when the user wants to backup their wallet.
+
+2. **Secure Storage**: Users are advised to store their seed phrase securely. It is crucial to keep this seed phrase in a safe, physical format (such as written on paper) and stored in a secure location to prevent unauthorized access or loss. The best practices in the [Backup & recovery]({{ '/guide/daily-spending-wallet/backup-and-recovery/landing-page/' | relative_url }}) section also apply to Cashu wallets. 
+
+{% include tip/open.html color="blue" icon="info" label="Mint Information Needed" %}
+
+Keep a record of the mints you are connected to. During the restoration process, you will need to reconnect to these specific mints to re-verify your tokens. Without this information, you will not be able to fully restore your wallet.
```

Comment:
Updated to
_"Instruct the user to keep a record of the mints or federations they are connected to. This can be done by writing them down or, since the mint list is not sensitive information, the wallet can offer to save the list of mints on cloud storage. During the restoration process, users will need to reconnect to these specific mints or federations to re-verify their ecash. Without this information, they will not be able to fully restore their wallet."_